hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/table/action/compact/SparkScheduleCompactionActionExecutor.java

```diff
@@ -85,4 +88,15 @@ protected HoodieCompactionPlan scheduleCompaction() {
     }
   }
 
+  protected boolean timeCompaction(String instantTime, String initialTime, int deltaCommitsSinceLastCompaction) {
+    if (Long.parseLong(initialTime) + config.getInlineCompactDeltaElapsedTimeMax() > Long.parseLong(instantTime)) {
```

Comment:
The timestamp is of `yyyyMMddHHmmss` format, we can not simply sum them by `+`

Reply:
revised